consensus/src/network_tests.rs

```diff
@@ -416,6 +484,44 @@ impl DropConfig {
     }
 }
 
+/// Table of per round message dropping rules
+struct DropConfigRound(HashMap<u64, DropConfig>);
+
+impl DropConfigRound {
+    /// Check if the message from 'src' to 'dst' should be dropped in the given round
+    pub fn is_message_dropped(&self, src: &TwinId, dst: &TwinId, round: u64) -> bool {
+        let mut result = false;
+
+        if self.0.contains_key(&round) {
```

Comment:
self.0.get(&round).and_then(|config| config.get(src).and_then(|set| set.contains(dst)).or_else(false)

Reply:
That looks better + concise, indeed! Done (with slight modification):

        self.0
            .get(&round)
            .and_then(|config| config.0.get(src)
                .and_then(|set| Some(set.contains(dst))))
            .or_else(|| {None})
            .is_some()